it/uk/gov/hmrc/agentaccesscontrol/MtdVatAuthorisationISpec.scala

```diff
@@ -0,0 +1,144 @@
+
+        status shouldBe 401
+      }
+
+      "there is no relationship between the agency and client" in {
+        given().agentAdmin(agentCode).isLoggedIn()
+          .andHasHmrcAsAgentEnrolment(arn)
+        given().mtdAgency(arn)
+          .hasNoRelationshipWith(vrn)
+
+        val status = authResponseFor(agentCode, vrn, method).status
+
+        status shouldBe 401
+      }
+    }
+
+    "send an AccessControlDecision audit event" in {
+      given().agentAdmin(agentCode).isLoggedIn()
+        .andHasHmrcAsAgentEnrolment(arn)
+      given().mtdAgency(arn)
+        .hasARelationshipWith(vrn)
+
+      authResponseFor(agentCode, vrn, method).status shouldBe 200
+
+      DataStreamStub.verifyAuditRequestSent(
+        AgentAccessControlDecision,
+        Map("path" -> s"/agent-access-control/mtd-vat-auth/agent/$agentCode/client/${vrn.value}"))
+    }
+
+    "record metrics for access control request" in {
+      given().agentAdmin(agentCode).isLoggedIn()
+        .andHasHmrcAsAgentEnrolment(arn)
+      given().mtdAgency(arn)
+        .hasARelationshipWith(vrn)
+      givenCleanMetricRegistry()
+
+      authResponseFor(agentCode, vrn, method).status shouldBe 200
+
+      timerShouldExistsAndBeenUpdated("API-Agent-MTD-VAT-Access-Control-GET")
+    }
+  }
+
+  "POST /agent-access-control/mtd-vat-auth/agent/:agentCode/client/:mtdItId" ignore {
+    val method = "POST"
+    "grant access when the agency and client are subscribed to the appropriate services and have a relationship" in {
+      given().agentAdmin(agentCode).isLoggedIn()
+        .andHasHmrcAsAgentEnrolment(arn)
+      given().mtdAgency(arn)
+        .hasARelationshipWith(vrn)
+
+      val status = authResponseFor(agentCode, vrn, method).status
+
+      status shouldBe 200
+    }
+
+    "not grant access" when {
+      "the agency is not subscribed to the appropriate service" in {
+        given().agentAdmin(agentCode).isLoggedIn()
+          .andHasNoHmrcAsAgentEnrolment()
+
+        val status = authResponseFor(agentCode, vrn, method).status
+
+        status shouldBe 401
+      }
+
+      "there is no relationship between the agency and client" in {
+        given().agentAdmin(agentCode).isLoggedIn()
+          .andHasHmrcAsAgentEnrolment(arn)
+        given().mtdAgency(arn)
+          .hasNoRelationshipWith(vrn)
+
+        val status = authResponseFor(agentCode, vrn, method).status
+
+        status shouldBe 401
+      }
+    }
+
+    "send an AccessControlDecision audit event" in {
+      given().agentAdmin(agentCode).isLoggedIn()
+        .andHasHmrcAsAgentEnrolment(arn)
+      given().mtdAgency(arn)
+        .hasARelationshipWith(vrn)
+
+      authResponseFor(agentCode, vrn, method).status shouldBe 200
+
+      DataStreamStub.verifyAuditRequestSent(
+        AgentAccessControlDecision,
+        Map("path" -> s"/agent-access-control/mtd-vat-auth/agent/$agentCode/client/${vrn.value}"))
+    }
+
+    "record metrics for access control request" in {
```

Comment:
May need to temporarily "ignore" this test for the metrics, as it might incur the same problems with the tests failing on Jenkins (which would be fixed in APB-2002)

Reply:
We are ignoring the full suite for now. Should that be fine?